book_source/02_demos_tutorials_workflows/01_install_pecan.Rmd

```diff
@@ -104,7 +122,7 @@ This will not go into much detail about about how to use Docker -- for more deta
 5. **Start the PEcAn stack**. Assuming all of the steps above completed successfully, start the full stack by running the following shell command:
 
     ```bash
-    docker-compose -p pecan up -d
+    docker-compose up -d
```

Comment:
this was here so we have a consistent name for the volumes and network, if not specified it will default to the folder where the docker-compose file is.

Reply:
See my comment above. That said, this is an easy change, so I'm happy to change it back.